app/services/push_notification/http_push.rb

```diff
@@ -91,7 +93,8 @@ def agency_uuid(service_provider)
         user_id: event.user.id,
         agency_id: service_provider.agency_id,
       )&.uuid ||
-        ServiceProviderIdentity.find_by(
+        # This changes breaks nothing
+        ServiceProviderIdentity.consented.find_by(
           user_id: event.user.id,
           service_provider: service_provider.issuer,
         )&.uuid
```

Comment:
honestly this code could probably be refactored to use `AgencyIdentity.for` instead of redoing the logic here